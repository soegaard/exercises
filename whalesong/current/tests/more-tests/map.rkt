#lang planet dyoo/whalesong

(map (lambda (x y) (+ x y)) (list 1 2 3) (list 4 5 6))
(map + (list 1 2 3) (list 4 5 6))
(map + (list 1 2 3) (list 4 5 6) (list 7 8 9))
(map + (list 1 2 3))
