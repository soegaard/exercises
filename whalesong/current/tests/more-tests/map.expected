(5 7 9)
(5 7 9)
(12 15 18)
(1 2 3)
